src/main/java/com/playonlinux/domain/Localisation.java

```diff
@@ -19,14 +19,21 @@
 package com.playonlinux.domain;
 
 import com.playonlinux.app.PlayOnLinuxContext;
+import com.playonlinux.domain.lang.LanguageBundle;
+import com.playonlinux.domain.lang.LanguageBundleSelector;
 import com.playonlinux.injection.Inject;
 import com.playonlinux.injection.Scan;
 import com.playonlinux.utils.ReplacableProperties;
+import java.util.Locale;
 
-
-// TODO
 @Scan
 public final class Localisation {
+    private static final LanguageBundle DEFAULT_BUNDLE;
+
```

Comment:
I think you can declare this bean in PlayOnLinuxConfig class file and use @Inject annotation